homepage/migrations/0008_alter_course_fields.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2 on 2021-04-27 14:21
+
+import django.core.validators
```

Comment:
```suggestion
from django.core import validators
```
in this way, you can use the validator directly without the django.core as prefix

Reply:
this is how the file is auto generated by Django but I'll change it